Ruby sort

aliciapaz opened the pull request:
This is the first project of Ruby module at Microverse.

The method allows to sort an array. It has two versions:
- `bubble_sort` allows to sort the array in ascending order
- `bubble_sort_by` allows to sort the array using a block provided when the method is called. This is possible using the `yield` keyword of Ruby.

jpdf00 committed:
Create bubble_sort.rb

Create file

jpdf00 committed:
Create .rubocop.yml

Create linters file

jpdf00 committed:
Update bubble_sort.rb

Finished code

jpdf00 committed:
Update bubble_sort.rb

Finish the project

aliciapaz committed:
add bubble_sort_yield method

jpdf00 committed:
Create bubble_sort_by.rb

Finish bubble_sort_by code

aliciapaz committed:
complete README.md

jpdf00 committed:
Create LICENSE

Add MIT License

aliciapaz committed:
update MIT license link in README

mahmoud717 reviewed (changes requested):
### Changes required ♻️ 

**Hello @aliciapaz  & @jpdf00**

Great job with the project so far it looks amazing, but there are still some changes to be made:
 
---
### Requested 🔴 
Your code is working, but:

- [x] Please make sure to fix all of your linters errors.
- [x] Please move you bubble_sort_by method to the bubble_sort.rb file and delete the other one.

---

please make these changes and request another code review.

if you have any questions don't hesitate to contact me on slack **MahmoudMohammad**

jpdf00 committed:
Update bubble_sort.rb

Fix Linters
Transfer code from bubble_sort_by to bubble_sort.rb
Delete file bubble_sort_by.rb

jpdf00 committed:
Update bubble_sort.rb

Fix linters on line 36

jpdf00 committed:
Update bubble_sort.rb

Fix linters on line 6 ans on line 40

aliciapaz committed:
fix linters and remove bubble_sort_by

aliciapaz committed:
Merge branch 'ruby_sort' into fix/linters

aliciapaz committed:
Merge pull request #2 from jpdf00/fix/linters

fix linters and remove bubble_sort_by

aliciapaz committed:
add line at the end

aliciapaz commented:
- Fix linters configuration and linter errors.
- Remove bubble_sort_by.rb

jubaan reviewed (approved):
# :tada: :tada: :tada: :rotating_light: Status: Approved :rotating_light:  :tada: :tada: :tada:

#### :clap: :clap: Great job :hammer_and_pick:!! There are no more issues left to be addressed. 

## See you at the top :rocket:!

# :tada: :tada: :tada: :rotating_light: Status: Approved :rotating_light:  :tada: :tada: :tada:
